recipes/divemesh/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "divemesh" %}
+{% set version = "23.08" %}
+{% set sha256 = "d67c287793da9972e6d4a4dd0cc281816382ea325a308ef434522e8d71993ed4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/REEF3D/DIVEMesh/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+  patches:
+    - patches/0001-remove-cxx-in-makefile.patch
+
+build:
+  number: 0
+  skip: true  # [win]
+  ignore_run_exports_from:
+    - {{ compiler('cxx') }}  # [osx]
```

Comment:
Why? This is compiled code amd thus should need this.

Reply:
Ahh, I remember now. @chenghlee recommended I try building with `--error-overlinking` and `--error-overdepending`, which raised this:
```    
conda_build.exceptions.OverDependingError: overdepending check failed 
    ['  ERROR (divemesh): dso library package conda-forge::libcxx-16.0.6-h4653b0c_0 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to `build/ignore_run_exports`)']
```

Reply:
Ignoring run_exports from a compiler is fine if you're doing it to avoid depending on unused standard libraries.